daemon/main.go

```diff
@@ -355,6 +357,8 @@ func init() {
 		option.AutoIPv6NodeRoutesName, false, "Automatically adds IPv6 L3 routes to reach other nodes for non-overlay mode (--device) (BETA)")
 	flags.StringVar(&bpfRoot,
 		"bpf-root", "", "Path to BPF filesystem")
+	flags.StringVar(&cgroupRoot,
+		"cgroup-root", "", "Path to Cgroup2 filesystem")
```

Comment:
Can we define a default? Ideally via the `pkg/defaults` package.

Reply:
modeled after bppfs I have this in defaults.go
```
        // DefaultCgroupRoot is the default path where cilium cgroup2 should be mounted
        DefaultCgroupRoot = "/mnt/cilium_cgroup2/"
```
If the user gives command line we will use it otherwise the above is used.

Reply:
per below comment though changed this to /var/run/cilium/cgroupv2

Reply:
@tgraf above look good?